djangocms_versioning/templates/djangocms_versioning/admin/duplicate_confirmation.html

```diff
@@ -0,0 +1,29 @@
+{% extends "admin/base_site.html" %}
+{% load i18n admin_urls static %}
+{% block title %}{% trans "Duplicate confirmation" %}{% endblock %}
+
+{% block extrahead %}
+    {{ block.super }}
+    {{ media }}
+    <script type="text/javascript" src="{% static 'admin/js/cancel.js' %}"></script>
+{% endblock %}
+
+{% block breadcrumbs %}{% endblock %}
+{% block bodyclass %}{{ block.super }} app-{{ opts.app_label }} model-{{ opts.model_name }} delete-confirmation{% endblock %}
+
+{% block content %}
+<p>{% trans "Are you sure you want to duplicate the following page?" %}</p>
```

Comment:
These template files appear to be page specific, They should reflect that somehow either the folder they are in or the template filename

Reply:
Called it duplicate_page_confirmation